editor/gui/editor_run_bar.cpp

```diff
@@ -425,6 +486,42 @@ EditorRunBar::EditorRunBar() {
 	main_hbox = memnew(HBoxContainer);
 	main_panel->add_child(main_hbox);
 
+	if (Engine::get_singleton()->is_recovery_mode_hint()) {
+		recovery_mode_popup = memnew(AcceptDialog);
+		recovery_mode_popup->set_min_size(Size2(550, 70));
```

Comment:
```suggestion
		recovery_mode_popup->set_min_size(Size2(550, 70) * EDSCALE); //needs #include "editor/themes/editor_scale.h" added
```
Otherwise, it will have strange size on hiDPI displays:
<img width="286" alt="Screenshot 2025-01-02 at 20 25 11" src="https://github.com/user-attachments/assets/60b0d508-83d7-40e7-8b29-c057edb7b80e" />

Reply:
Fixed, thanks :+1: